recipe/meta.yaml

```diff
@@ -56,8 +59,7 @@ requirements:
     - pkg-config
     # cross-compiling:
     - cross-python_{{ target_platform }}  # [build_platform != target_platform]
-    - numpy                               # [build_platform != target_platform and (py<39 or py>=311)]
-    - numpy 1.22                          # [build_platform != target_platform and 39<=py<311]
+    - numpy >=2.0.0a0                     # [build_platform != target_platform]
```

Comment:
Why a0 and not rc1?

Reply:
It's more general, but no very good reason.

Reply:
OK, according to the Numpy migration guide, this should be rc1.

Reply:
```suggestion
    - numpy >=2.0.0rc1                    # [build_platform != target_platform]
```